src/menu/views/settings_editor.c

```diff
@@ -112,10 +158,14 @@ static component_context_menu_t set_rumble_enabled_type_context_menu = { .list =
 #endif
 
 static component_context_menu_t options_context_menu = { .list = {
+    { .text = "ROM Loading Bar", .submenu = &set_loading_progress_bar_enabled_context_menu },
```

Comment:
This should not be available in the UI settings editor. It was added for a specific use case where users that required autoload and had scalers had issues with the flipping of resolution. The autoload function is now deprecated and due to be removed as we now have fast reboot.